vcf/filter.go

```diff
@@ -27,59 +28,74 @@ func Filter(v *Vcf, chrom string, minPos int64, maxPos int64, ref string, alt st
 	return true
 }
 
+//FilterQual returns true if a Vcf quality is above an input minQual score, false otherwise.
 func FilterQual(v *Vcf, minQual float64) bool {
 	if v.Qual < minQual {
 		return false
 	}
 	return true
 }
 
-func FilterAlt(v *Vcf, alt string) bool {
-	if alt != "" && v.Alt != alt {
+//FilterAlt returns true if the Alt field of a Vcf entry matches a desired input Alt field, false otherwise
+func FilterAlt(v *Vcf, alt []string) bool {
```

Comment:
Would a name like "AltMatches" or "AltEquals" be better?  It looks like an empty alt being passed in would match everything?  It also looks like the alt fields need to be in the same order, so C,G does not equal G,C?  Maybe a comment that order matters?

Reply:
I'll keep the name consistent with the other filter functions for now but add a comment that order matters.